backend/tests/Functional/NelmioApiDocsTest.php

```diff
@@ -11,6 +11,6 @@ final class NelmioApiDocsTest extends WebTestCase
     public function testApiDoc(): void
     {
         self::$client->request('GET', '/api/doc');
-        self::assertResponseStatusCodeSame(200);
+        self::assertResponseIsSuccessful();
```

Comment:
```suggestion
        self::assertResponseStatusCodeSame(200);
```
.. is more strictly checking for the correct response